gatsby-theme-oi-wiki/src/components/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import {Typography} from '@material-ui/core'
+
+import {makeStyles} from '@material-ui/core/styles'
+import {graphql, useStaticQuery} from 'gatsby'
+import React, {useEffect} from 'react'
+import {SmartLink} from './Link'
```

Comment:
```suggestion
import {Typography} from '@material-ui/core'

import {makeStyles} from '@material-ui/core/styles'
import {graphql, useStaticQuery} from 'gatsby'
import React, {useEffect} from 'react'
import {SmartLink} from './Link'
```

这个地方的大括号周围的空格不见了，revert 力度过大了

Reply:
@Enter-tainer 是现在的 eslint 配置 fix 的（不是我改的，是默认的配置），之后再统一处理一下，改一些配置？还可以考虑是不是要用 prettier 之类的？

Reply:
但是别的文件里面的大括号周围都是有空格的，是不是可以统一一下

Reply:
@Enter-tainer 那请问需要我现在就给 eslint 加上添加空格的配置吗？

Reply:
好的，可以加

Reply:
@Enter-tainer 搞定啦